pkgs/by-name/tt/tt/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec{
+  pname = "tt";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "lemnos";
+    repo = "tt";
+    rev = "refs/tags/v${version}";
+    sha256 = "oe0gegMINVmbw1UPvSy9G67D4amQTumlrgvb8H+Y4IM=";
```

Comment:
Any specific reason why `sha256` is used rather than the preferred to `hash` with a `sha256-…` prefixed value (as is done with `vendorHash`)?

Reply:
absolutely not, it just worked so i did that but i'm gonna change that, thanks 👍